cluster/engine_test.go

```diff
@@ -222,14 +222,21 @@ func TestEngineSpecs(t *testing.T) {
 	assert.True(t, engine.isConnected())
 	assert.True(t, engine.IsHealthy())
 
-	assert.Equal(t, engine.Cpus, int64(mockInfo.NCPU))
-	assert.Equal(t, engine.Memory, mockInfo.MemTotal)
-	assert.Equal(t, engine.Labels["storagedriver"], mockInfo.Driver)
-	assert.Equal(t, engine.Labels["executiondriver"], mockInfo.ExecutionDriver)
-	assert.Equal(t, engine.Labels["kernelversion"], mockInfo.KernelVersion)
-	assert.Equal(t, engine.Labels["operatingsystem"], mockInfo.OperatingSystem)
+	mockInfo2 := mockInfo
+	mockInfo2.Labels = []string{"foo=bar", "executiondriver=newdriver", "node=node1"}
+
+	assert.Equal(t, engine.Cpus, int64(mockInfo2.NCPU))
+	assert.Equal(t, engine.Memory, mockInfo2.MemTotal)
+	assert.Equal(t, engine.Labels["storagedriver"], mockInfo2.Driver)
+
+	assert.Equal(t, engine.Labels["executiondriver"], "execution-driver-test")
```

Comment:
@allencloud shouldn't this be `mockInfo2.ExecutionDriver`, to make sure it's future proof?